JASP-Engine/JASP/R/commonbayesianttest.R

```diff
@@ -1880,18 +1883,18 @@
       priorUpper <- qnorm(upperp, options[["informativeNormalMean"]], options[["informativeNormalStd"]])
 
       # compute 95% credible interval & median:
-      ci95PlusMedian <- .ciPlusMedian_normal(t = t, n1 = n1, n2 = n2, independentSamples = ! paired && !is.null(n2),
-                                             prior.mean = options[["informativeNormalMean"]],
-                                             prior.variance = options[["informativeNormalStd"]]^2,
-                                             ci = .95, oneSided = oneSided)
-      CIlow <- ci95PlusMedian[["ciLower"]]
-      CIhigh <- ci95PlusMedian[["ciUpper"]]
-      medianPosterior <- ci95PlusMedian[["median"]]
+      ciPlusMedian <- .ciPlusMedian_normal(t = t, n1 = n1, n2 = n2, independentSamples = ! paired && !is.null(n2),
+                                           prior.mean = options[["informativeNormalMean"]],
+                                           prior.variance = options[["informativeNormalStd"]]^2,
+                                           ci = ci.level, oneSided = oneSided)
+      CIlow <- ciPlusMedian[["ciLower"]]
+      CIhigh <- ciPlusMedian[["ciUpper"]]
+      medianPosterior <- ciPlusMedian[["median"]]
 
     }
 
-    xlim[1] <- min(-2, ci99PlusMedian[["ciLower"]], priorLower)
-    xlim[2] <- max(2, ci99PlusMedian[["ciUpper"]], priorUpper)
+    xlim[1] <- min(-2, ci99PlusMedian[["ciLower"]], CIlow,  priorLower)
```

Comment:
would it be reasonable to have here ciLower as well instead of CIlow, it suggests that we are using different names for the same thing. This most likely due to many different people working on this before. 

Same with CIhigh <- also note it's not properly camelCased

Reply:
Yeah I mean... I can do that. The problem is, there is really no consistency all over the t-tests, if you want to pick on `CIhigh`, we might as well just rewrite pretty much most of the code.

Reply:
What I mean by this is that I am happy to change `ci.level` to `ciValue` since that is a name that I introduce in this PR, so it's fine to change to camelCase right now. For the other names, I just reuse what is already in the code, and it seems arbitrary to then rename only `CIlow` and not, for example, stuff like `prior.location`. So I would prefer to skip these code consistency changes in this PR (which is about new functionality, not code consistency), and just make a separate issue/PR for renaming everything properly from scratch...

Reply:
(I must admit that the other reason of course is that I don't wanna do go over these things right now 😅 , sorry if I sound too defensive)